kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/form/render/AbstractFormRenderer.java

```diff
@@ -700,19 +726,28 @@ protected void appendFieldJSON(StringBuilder jsonTemplate, String type, String n
                 .append(name)
                 .append("' : ")
                 .append(getJSFieldType(type))
-                .append(appendExtractionExpression(type, name, id, jsType, false))
+                .append(appendExtractionExpression(type, name, id, jsType, false,""))
                 .append(wrapEndFieldType(type))
                 .append(",");
     }
 
     protected void appendFieldJSON(StringBuilder jsonTemplate, String type, FormField field, String jsType) {
-        jsonTemplate.append("'")
-                .append(field.getBinding())
-                .append("' : ")
-                .append(getJSFieldType(type))
-                .append(appendExtractionExpression(type, field.getBinding(), field.getId(), jsType, field.isShowTime()))
-                .append(wrapEndFieldType(type))
-                .append(",");
+        if ("datetime-local".equals(type) || "Date".equals(type)
```

Comment:
Would it be possible to turn the condition into a boolean method instead? Some distinctive naming that will allow using the method also in `appendExtractionExpression` as the logic seems to be the same.

Reply:
Ad my comment about changing case elsewhere - is this capital letter in `Date` the actual reason?